src/styles/main.scss

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  width: $calendar-size;
+  display: flex;
+  padding: $padding;
+  justify-content: left;
+  flex-wrap: wrap;
+  gap: $block-gap;
+  font-size: 30px;
+
```

Comment:
The '.calendar' block has a 'width' property that relies on a variable '$calendar-size' which is not defined within the provided code. Ensure that the variable is defined in the 'variables.scss' file or within this file if it's missing.